src/main/resources/CreateDB.sql

```diff
@@ -0,0 +1,63 @@
+USE Team2_CallumE;
+
+CREATE TABLE Auth_Roles(
+	RoleID SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT,
+    Role_Name VARCHAR(70) NOT NULL,
+    PRIMARY KEY(RoleID)
+);
+
+CREATE TABLE Users(
```

Comment:
There's no uniqueness in users; I can use the same email address multiple times and not know which one to check the password against